atom/browser/api/atom_api_window.cc

```diff
@@ -784,9 +784,15 @@ bool Window::IsModal() const {
 }
 
 v8::Local<v8::Value> Window::GetNativeWindowHandle() {
```

Comment:
This function is exposed in BrowserWindow but it isn't use at all by `browser-laptop`, can we remove it?